community/office-hours.md

```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
 Tune into the [Kubernetes YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/live) to follow along. 
 You can post questions on the [#office-hours channel](https://kubernetes.slack.com/messages/office-hours) on Slack.
 
+### How it Works
+
+#### Bringing your own Stack Overflow Question
+
+If you submit a [SO question](https://stackoverflow.com/questions/tagged/kubernetes) we can prepare ahead of time and check important details. 
+
+As a thanks to the community the asker can then ensure that a well written answer makes it way to the question afterwards so that we can build a knowledge base and help maintain the incoming questions on SO. We can also use the video archive of each meeting to bring context to each SO question we answer. 
```

Comment:
where would the final log of question/answer go? meeting notes? .md file in a repo someplace?

Reply:
Ideally in the SO question itself.